Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionControllerCocoa.mm

```diff
@@ -570,13 +573,16 @@ - (void)_didDeleteLocalStorage:(NSNotification *)notification
     return FileSystem::pathByAppendingComponent(m_configuration->storageDirectory(), uniqueIdentifier);
 }
 
-_WKWebExtensionStorageSQLiteStore *WebExtensionController::sqliteStore(const String& storageDirectory, WebExtensionDataType type, RefPtr<WebExtensionContext> extensionContext)
+RefPtr<WebExtensionStorageSQLiteStore> WebExtensionController::sqliteStore(const String& storageDirectory, WebExtensionDataType type, RefPtr<WebExtensionContext> extensionContext)
 {
-    if (type == WebExtensionDataType::Session)
-        return extensionContext ? extensionContext->storageForType(WebExtensionDataType::Session) : nil;
+    if (type == WebExtensionDataType::Session) {
+        if (extensionContext)
+            return extensionContext->storageForType(WebExtensionDataType::Session);
+        return nullptr;
+    }
 
     auto uniqueIdentifier = FileSystem::lastComponentOfPathIgnoringTrailingSlash(storageDirectory);
-    return [[_WKWebExtensionStorageSQLiteStore alloc] initWithUniqueIdentifier:uniqueIdentifier storageType:type directory:storageDirectory usesInMemoryDatabase:NO];
+    return WebExtensionStorageSQLiteStore::create(uniqueIdentifier, type, storageDirectory, false);
```

Comment:
The `false` param here really should be an `enum class : bool` like `UsesInMemoryDatabase::No` so it is readable inline.